.github/actions/install-deps/action.yaml

```diff
@@ -2,20 +2,22 @@ name: Install dependencies
 description: install all project dependencies.
 
 inputs:
-  node-version:
+  node_version:
```

Comment:
How come we're proposing snake casing for input keys? It's a breaking change to this action and creates inconsistency between other inputs in this action and all third-party actions pretty much - which all use kebab casing. Even GitHub use kebab casing for all their YAML keys.

Reply:
Yup, our other workflows used snake and kabab base interchangably, but the majority were snake case. @AnnaDodson and I made a snap decission in the office on Wednesday to adopt snake case for out own actions at least. At the time also, I'd thought this also has the benefit of distincting between our inputs/variables and third-party, for example: `node-version: ${{ inputs.node_version }}` reads to me as setting node-version in a third party action from a input/var from our own.

Reply:
I'm not sure I understand why we need a distinction between DVSA and third-party inputs. 

The above example would not be used in a workflow used by other projects, that's an internal mapping to a third-party action that uses kebab-casing in their inputs in this case `setup-node`. 

Project workflows are also quite likely to mix in some third-party workflows, this would cause the workflows themselves to become inconsistent.

For example:
```yaml
use: dvsa_snake_case
with:
  node_version: '18'
  other_input: 'A'

use: release-please
with:
   release-type: node
   package-name: release-please-action
```
# vs 
```yaml
use: dvsa-kebab-case
with:
  node-version: '18'
  other-input: 'A'

use: release-please
with:
   release-type: node
   package-name: release-please-action
```

I'll resolve this anyhow as it's only my opinion - I'm happy either way.

Reply:
Could be one for the team meeting Monday?

Reply:
@sdh100shaun , can we have some time for this on Monday Dev Team meeting?